packages/ecs-agent/ecs.config

```diff
@@ -1,2 +1,6 @@
 ECS_LOGFILE=/var/log/ecs/ecs-agent.log
 ECS_LOGLEVEL="{{settings.ecs.loglevel}}"
+{{#if settings.network.http-proxy}}
+HTTP_PROXY={{settings.network.http-proxy}}
+{{/if}}
+NO_PROXY={{#each settings.network.no-proxy}}{{#if @index}},{{/if}}{{this}}{{/each}}{{#if settings.network.no-proxy}},{{/if}}169.254.169.254,169.254.170.2,/var/run/docker.sock
```

Comment:
`/var/run/docker.sock` should also just be `/run/docker.sock` because that's where the docker socket is in Bottlerocket

Reply:
`/var/run` is a symlink to `/run`.  The ECS agent [expects `/var/run/docker.sock`](https://github.com/aws/amazon-ecs-agent/blob/1489adfaec05c559ea4e1490774d1f86261cf936/agent/config/config_unix.go#L48), so this value is correct.

Reply:
Nevermind, `/var/run` is symlinked to `/run`, so this is correct as is